functions/src/modules/CR.functions.ts

```diff
@@ -1,7 +1,14 @@
 import * as functions from 'firebase-functions';
+import * as admin from 'firebase-admin';
 import * as helpers from '../helpers';
 import GoogleSheets from '../services/GoogleSheets';
 import * as _ from 'lodash';
+import * as moment from 'moment';
+
+export enum ISoundQualityReportSheet {
```

Comment:
Sound Quality Report is not related to the Content Reporting. I'd suggest something like
```suggestion
export enum SheetNames {
```